examples/simple/project.clj

```diff
@@ -0,0 +1,5 @@
+(defproject cljsbuild-example-simple "1.1.7"
```

Comment:
Not sure why this is `cljsbuild-example-simple`, since there's no `cljsbuild` involved (that I can see). Maybe `views-example-simple`?

Reply:
Good catch. Mixed with another example project I was working on